code-challenges.test.js

```diff
@@ -26,27 +36,78 @@ var num3 = -7
 
 // b) Create the function that makes the test pass.
 
+//create function named divByThree
+//function takes 1 paramater, num
+//use a conditional and the modulo operator to get desired output
+//run yarn jest
+
+const divByThree = (num) => {
+  if (num % 3 === 0) {
+    return num + ' is divisible by three'
+  } else {
+    return num + ' is not divisible by three'
+  }
+}
+
+//originally got 1 failed
+//reason was missing '+' after last num.
+//corrected, and passed
+
 
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("capWords", () => {
+  it("returns an array with all the words capitalized", () => {
+    expect(capWords(["streetlamp", "potato", "teeth", "conclusion", "nephew"])).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(capWords(["temperature", "database", "chopsticks", "mango", "deduction"])).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+  })
+})
+
 var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
-
+//$ yarn jest. 1 failed, function not defined
 
 // b) Create the function that makes the test pass.
 
+//create capWords function
+//iterate over array using map
+//use map to change first index use to uppercase to change the first letter
+//return result arrays
+
+const capWords = (array) => {
+
+  let words = array.map(value => {
+// here I realized I need to access the zero index then add the rest of the word
+    return value[0].toUpperCase() + value.substring(1)
+
+  })
+  return words
+}
+
```

Comment:
Nice job, the logic looks lovely. The only refactor I would offer is if you are creating a variable just to return it, you can skip creating the variable and just return the statement.